source/admin_manual/list_of_events.rst

```diff
@@ -2601,6 +2601,24 @@ indexes in metacache.
 | error                    | Error message if the rescan/rebuild failed           |
 +--------------------------+------------------------------------------------------+
 
+
+obox_save_throttling
+^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.4.0;3.0.0
+
+Obox is throttling the number of concurrent saves/copies (this is an event used
+in tests)
```

Comment:
Maybe this can mention the related settings.

So saying that `mail_prefetch_count` and `obox_max_parallel_writes` cannot be reached in some situation (mainly dsync?) and in that cases this event is emitted?